pset-functions.js

```diff
@@ -147,7 +187,19 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 4:')
 
 // Add your code below this line
-
+const doubleLetters = word => {
+    for(i=0 ; i <= word.length ; i++){
+        for (let j = i +1 ; j <= word.length; j++ ){
+            if(word[j] === word[i]){
```

Comment:
This solution definitely works, but nested loops can cause your code to run slow (try to avoid it where you can). Instead, you can take advantage of the first for loops iteration and bracket notation like in the example [here](https://github.com/jsd20200127/hw-02-functions-solution/blob/e87c29073cf7285a5045ec8708561f279474f2d2/pset-functions.js#L225).